common/app/common/contexts/ProjectContext.java

```diff
@@ -22,6 +26,14 @@ private ProjectContext(final List<Locale> languages, final List<CountryCode> cou
         return countries;
     }
 
+    public Locale defaultLanguage() {
+        return languages.stream().findFirst().get();
```

Comment:
I can't it hurts my eyes :P
I put it like this because I prefer how it looks like and theoretically it should do the same without performance loss.